Covid/Services/LocationServices/BeaconManager.swift

```diff
@@ -75,6 +76,12 @@ final class BeaconManager: NSObject {
             ]
         )
     }
+    
+    func activateLocationTrackingForBeacons() {
+        locationManager.requestAlwaysAuthorization()
```

Comment:
out-of-the-scope koment:

vsimol som si, ze nehandlujeme niektore, edge casey, napr. ak OS killne appku a potom ju nasledne nastartatuje na zaklade location eventu, tento even nespracovavame..
po release sa na to mozme pozriet.

eg

** UIApplicationLaunchOptionsLocationKey**
_A key indicating that the app was launched to handle an incoming location event._

_The value of this key is an NSNumber object containing a Boolean value. You should use the presence of this key as a signal to create a CLLocationManager object and start location services again. Location data is delivered only to the location manager delegate and not using this key._

zdroj
https://developer.apple.com/documentation/uikit/uiapplicationlaunchoptionslocationkey

Reply:
dobbre vediet, snazime sa to spravit pri akomkolvek spusteni :D

Reply:
prave som mrkol, ako sme to riesili pred 5 rokmi v Downtowne, toto riesenieuz moze byt dost outdated, no mozno by nebolo od veci ho oskusat :)

```
    if (launchOptions[UIApplicationLaunchOptionsLocationKey]) {
        // [_locationManager startMonitoringSignificantLocationChanges]
        // [_beaconManager startMonitoringForRegion: ...]
    }
```